examples/digital_fingerprinting/production/mlflow/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt update && \
     rm -rf /var/cache/apt/* /var/lib/apt/lists/*
 
 # Install python packages
-RUN pip install "mlflow >=2.2.1,<3" boto3 pymysql pyyaml
+RUN pip install "mlflow >=2.10.0,<3" boto3 pymysql pyyaml
```

Comment:
Lets make an issue to generate this with the dependency file generator so they stay synced long term

Reply:
Done issue #1615